bundles/org.openhab.binding.mail/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -40,22 +41,22 @@
 	</thing-type>
 	<thing-type id="imap" extensible="mailcount">
 		<label>IMAP Server</label>
-		<description>Used for receiving mail</description>
+		<description>Used for receiving mails</description>
```

Comment:
I think "mail" sounds better than "mails".

Reply:
Thanks - I've updated the texts accordingly.